File: app/Models/PengaturanGaji.php
<?php
namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class PengaturanGaji extends Model
{
    protected $table = 'pengaturan_gaji';

    protected $fillable = [
        'upah_harian_supir',
        'upah_harian_kernet',
        'tunjangan_makan_harian',
        'subsidi_transport_harian',
        'komisi_muatan_berat_supir',
        'komisi_muatan_berat_kernet',
        'tarif_kir',
        'potongan_kesehatan_supir',
        'potongan_kesehatan_kernet',
        'jamsostek_supir',
        'jamsostek_kernet',
    ];
}

File: Models/PengaturanGaji.cs
using System.Text.Json.Serialization;

namespace SistemPengirimanApp.Models;

public class PengaturanGaji
{
    [JsonPropertyName("tunjangan_makan_harian")]
    public decimal TunjanganMakanHarian { get; set; }

    [JsonPropertyName("subsidi_transport_harian")]
    public decimal SubsidiTransportHarian { get; set; }

    [JsonPropertyName("potongan_kesehatan_supir")]
    public decimal PotonganKesehatanSupir { get; set; }

    [JsonPropertyName("potongan_kesehatan_kernet")]
    public decimal PotonganKesehatanKernet { get; set; }

    [JsonPropertyName("jamsostek_supir")]
    public decimal JamsostekSupir { get; set; }

    [JsonPropertyName("jamsostek_kernet")]
    public decimal JamsostekKernet { get; set; }
}